src/app/p-common/menubar/menubar.component.ts

```diff
@@ -9,17 +9,27 @@ import { MenuItem } from 'primeng/api';
 export class MenubarComponent implements OnInit {
 
   items: MenuItem[];
+  currentProject: string;
 
   constructor() {
+    this.currentProject = localStorage.getItem('papaya-project');
   }
 
   ngOnInit() {
     this.items = [
       {
-        label: 'Projekty',
-        icon: 'pi pi-fw pi-list',
+        icon: 'pi',
+        style: {
+          'background-image': 'url("../../../assets/papaya-logo-32-32.png")',
+          'background-repeat': 'no-repeat',
+          'background-position': 'center'
+        },
         routerLink: [ '/projects' ]
       },
+      {
+        label: this.currentProject,
+        routerLink: [ '/example' ]
```

Comment:
na pewno example? wydaje mi się że można już postawić ProjectsModule

Reply:
na pewno example - to jest tzw. "pusty ekran", acz przydałaby się zmiana nazwy modułu.
natomiast ProjectsModule już jest, i to jest ekran wyboru projektu (z przyciskiem do tworzenia nowego) - widoczny zaraz po zalogowaniu, lub po kliknięciu papaya-logo na menubarze.